guide/raids/amirdrassil.md

```diff
@@ -0,0 +1,445 @@
+* Use {{ site.data.spell.healing_surge }} if necessary.
+
+#### Utilities:
+* Use {{ site.data.talent.wrt }} to help players dealing with {{ site.data.raid.amirdrassil.polymorph_bomb }}.
+* Use {{ site.data.talent.pct }} to dispel players affected by {{ site.data.raid.amirdrassil.poisonous_javelin }} but keep in mind that you need to be spread to avoid suffering to {{ site.data.raid.amirdrassil.unstable_venom }}.
+
+</div>
+</div>
+</div>
+
+<div class="card">
+<div class="card-header" id="larodar">
+<div data-toggle="collapse" data-target="#larodar-collapse" aria-expanded="true" aria-controls="larodar-collapse" class="raid-header amirdrassil_the_dreams_hope5"><h2>Larodar, Keeper of the Flame</h2></div>
+</div>
+<div id="larodar-collapse" class="collapse" aria-labelledby="larodar" data-parent="#accordion">
+<div class="card-body" markdown="1">
+
+## Boss Fight Profile: Single Target / Cleave
+
+### Talents
+
+#### Lightning T30:
+<div class="iframe-holder">
+<iframe src="https://www.raidbots.com/simbot/render/talents/BYQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAoVSSLJUSSLRDRJQiEAAAAAgSAJlkgmASLJpECIRIBC?width=530&level=70" frameborder="0" width="530px" height="100%">
+</iframe>
+</div>
+
+#### Fire T31:
+<div class="iframe-holder">
+<iframe src="https://www.raidbots.com/simbot/render/talents/BYQAAAAAAAAAAAAAAAAAAAAAAAAAAAAg0SrUSCgDIlk0oEl0SCRAAAAAAKBkUSCaCItkkSKBIRCBC?width=530&level=70" frameborder="0" width="530px" height="100%">
+</iframe>
+</div>
+
+<hr>
+
+## Fight Breakdown
+
+#### Damage:
+* Focus down {{ site.data.raid.amirdrassil.fiery_treant }} and {{ site.data.raid.amirdrassil.ashen_treant }} whenever they are active.
+* Use {{ site.data.talent.swg }} to maintain uptime whilst dealing with {{ site.data.raid.amirdrassil.dream_blossom }}, {{ site.data.raid.amirdrassil.ashen_davastation }} (Mythic-only) or whilst kiting {{ site.data.raid.amirdrassil.scorching_roots }}.
+
+#### Defensives:
+* Use {{ site.data.talent.as }} or healing consumables during {{ site.data.raid.amirdrassil.raging_inferno }} and when targeted by {{ site.data.raid.amirdrassil.ashen_devastation }} (Mythic-only).
+* Use {{ site.data.talent.ag }} during {{ site.data.raid.amirdrassil.raging_inferno }} and when affected by {{ site.data.raid.amirdrassil.flash_fire }}.
+* If talented, use {{ site.data.talent.earth_shield }} and {{ site.data.talent.healing_stream_totem }} as a movement filler throughout the fight.
+* Use {{ site.data.spell.healing_surge }} if necessary.
+
+#### Utilities:
+* Use {{ site.data.talent.wrt }} to help players affected by {{ site.data.raid.amirdrassil.dream_blossom }} dousing {{ site.data.raid.amirdrassil.burning_ground }} as fast as possible.
+* Use {{ site.data.talent.spirit_walk }} to free yourself from {{ site.data.raid.amirdrassil.scorching_bramblethorn }}.
+* Use {{ site.data.spell.wind_shear }} or {{ site.data.spell.capacitor_totem }} to interrupt {{ site.data.raid.amirdrassil.fiery_flourish }} from {{ site.data.raid.amirdrassil.fiery_treant }}.
```

Comment:
site.data.talent.capacitor_totem and not site.data.spell.capacitor_totem

Maybe I will merge talent and spell files later to avoid those problems, they are just annoying...